charts/container-deployer/values.yaml

```diff
@@ -10,12 +10,15 @@ deployer:
 #  namespace: ""
   initContainer:
     repository: eu.gcr.io/gardener-project/landscaper/container-deployer-init
+#    tag: "latest"
   waitContainer:
     repository: eu.gcr.io/gardener-project/landscaper/container-deployer-wait
-  defaultImage: {}
+#    tag: "latest"
+#  defaultImage:
```

Comment:
`initContainer` and `waitContainer` have dedicated `respository` and `tag` fields. Shouldn't we have the same for `defaultImage`? Just for consistency? Of course that would require changing the `_helpers.tpl` as well...

Reply:
**Definitely needs a change in `_helpers.tpl`** which made the integration test fail. If `defaultImage` is commented out like in here,
```
{{- if .Values.deployer.defaultImage.image }}
defaultImage:
  image: "{{ .Values.deployer.defaultImage.image }}"
{{- end }}
```
in https://github.com/gardener/landscaper/blob/a1e3520394e8e244816e0c7d5c3c581d2c6c3a20/charts/container-deployer/templates/_helpers.tpl#L76 will `nil`.